brambling/models.py

```diff
@@ -378,14 +421,32 @@ def stripe_live_can_connect(self):
     def stripe_test_can_connect(self):
         return bool(stripe_test_settings_valid() and not self.stripe_test_user_id)
 
-    def get_invites(self):
-        return Invite.objects.filter(kind=Invite.ORGANIZATION_EDITOR,
-                                     content_id=self.pk)
-
     def is_demo(self):
         return self.slug == Organization.DEMO_SLUG
 
 
+class EventMember(models.Model):
+    EDIT = '1-edit'
+    VIEW = '2-view'
+    ROLE_CHOICES = (
+        (EDIT, 'Can edit event'),
+        (VIEW, 'Can view event'),
+    )
+    event = models.ForeignKey('Event')
+    person = models.ForeignKey('Person')
+    role = models.CharField(max_length=6, choices=ROLE_CHOICES)
```

Comment:
How strongly do you feel about using CharField here? My attitude is, if we're using postgres, to just put store strings in the `text` type and not have to worry about length enforcement. Especially considering this field doesn't accept user data, limiting ourselves to 6 characters feels stingy if we ever want to add more roles.